src/pages/masternodes/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { GetServerSidePropsContext, GetServerSidePropsResult, InferGetServerSidePropsType } from 'next'
+import { CursorPage, CursorPagination } from '@components/commons/CursorPagination'
+import { getWhaleApiClient } from '@contexts/WhaleContext'
+import { MasternodeData } from '@defichain/whale-api-client/dist/api/masternodes'
+import { Head } from '@components/commons/Head'
+import NumberFormat from 'react-number-format'
+import { OverflowTable } from '@components/commons/OverflowTable'
+
+interface MasternodesPageProps {
+  masternodes: {
+    items: MasternodeData[]
+    pages: CursorPage[]
+  }
+}
+
+export default function MasternodesPage ({ masternodes }: InferGetServerSidePropsType<typeof getServerSideProps>): JSX.Element {
+  return (
+    <div className='container mx-auto px-4 pt-12 pb-20'>
+      <Head title='Masternodes' />
+      <h1 className='text-2xl font-semibold'>Masternodes</h1>
+      <OverflowTable className='mt-6'>
+        <OverflowTable.Header>
+          <OverflowTable.Head sticky>OWNER</OverflowTable.Head>
+          <OverflowTable.Head>OPERATOR</OverflowTable.Head>
+          <OverflowTable.Head>CREATION HEIGHT</OverflowTable.Head>
+          <OverflowTable.Head>RESIGN HEIGHT</OverflowTable.Head>
+          <OverflowTable.Head>MINTED BLOCKS</OverflowTable.Head>
+          <OverflowTable.Head>STATE</OverflowTable.Head>
+        </OverflowTable.Header>
+        {masternodes.items.map((mn) => (
+          <MasternodeRow data={mn} key={mn.id} />
+        ))}
+      </OverflowTable>
+      <div className='flex justify-end mt-8'>
+        <CursorPagination pages={masternodes.pages} path='/masternodes' />
+      </div>
+    </div>
+  )
+}
+
+function MasternodeRow ({ data }: {data: MasternodeData}): JSX.Element {
+  return (
+    <OverflowTable.Row>
+      <OverflowTable.Cell sticky>
+        {data.owner.address}
+      </OverflowTable.Cell>
+      <OverflowTable.Cell>
+        {data.operator.address}
+      </OverflowTable.Cell>
```

Comment:
We need to wrap these into 2 lines. Otherwise, they are too long:
![image](https://user-images.githubusercontent.com/4266087/130177084-d062b5d0-eea8-4676-b71f-e07058d9f5c9.png)